studio/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
```

Comment:
@rishabhdas Could you please remove subcharts from `charts/` directory and add `*.tgz` to `.gitignore`? 

`Charts.lock`  is enough to store information about the dependencies.

Reply:
Implemented as requested